R/create_sample_histograms.R

```diff
@@ -0,0 +1,84 @@
+#' Create a sample histogram from a population
+#'
+#' This function creates a grid of sample distributions from a population for different sample sizes.
+#' Author: Holly Williams
+#'
+#' @param pop the virtual population as a tibble
+#' @param samples the samples as a tibble
+#' @param var_name the name of the column for the variable that is being generated
+#' @param n_s a vector of the sample sizes
+#'
+#' @return a list of the sample histogram plots
+#' @export
+#'
+#' @examples
+#' pop <- generate_virtual_pop(100, height, rnorm, 0, 1)
+#' samples <- draw_samples(pop, 3, c(1, 10))
+#' create_sample_histograms(pop, samples, height, c(1, 10))
+create_sample_histograms <- function(pop, samples, var_name, n_s){
+
+  # START INPUT TESTS #
+  # convert var_name to string for testing
+  col_name <- toString(get_expr(enquo(var_name)))
```

Comment:
The `get_expr()` function requires `rlang` package. You probably should use rlang::

Reply:
Good suggestion! Incorporated in latest commit